packages/performance/src/services/oob_resources_service.ts

```diff
@@ -119,3 +138,45 @@ function createUserTimingTrace(
   }
   Trace.createUserTimingTrace(performanceController, measureName);
 }
+
+function sendOobTrace(performanceController: PerformanceController): void {
+  if (!sentPageLoadTrace) {
+    sentPageLoadTrace = true;
+    const api = Api.getInstance();
+    const navigationTimings = api.getEntriesByType(
+      'navigation'
+    ) as PerformanceNavigationTiming[];
+    const paintTimings = api.getEntriesByType('paint');
+    // If First Input Delay polyfill is added to the page, report the fid value.
+    // https://github.com/GoogleChromeLabs/first-input-delay
+    if (api.onFirstInputDelay && !firstInputDelay) {
+      setTimeout(() => {
+        Trace.createOobTrace(
+          performanceController,
+          navigationTimings,
+          paintTimings,
+          webVitalMetrics,
+          firstInputDelay
+        );
+      }, FID_WAIT_TIME_MS);
```

Comment:
Is this needed anymore since we are always sending at the end of the page session. Since we call this on page unload, we don't need an extra timeout here.

Reply:
Done